api/v1/server/oas/transformers/v1/log.go

```diff
@@ -8,9 +8,27 @@ import (
 )
 
 func ToV1LogLine(log *sqlcv1.V1LogLine) *gen.V1LogLine {
+
+	retryCount := int(log.RetryCount)
+	attempt := retryCount + 1
```

Comment:
nit: might be good to factor this into a helper? I don't super duper care since it's so self-explanatory though 🤷

Reply:
generally would agree, i think this is fine though. bigger concern that i dont have a solve for is that we miss setting this on future transformers